Lib/unittest/loader.py

```diff
@@ -216,9 +216,7 @@ def shouldIncludeMethod(attrname):
             testFunc = getattr(testCaseClass, attrname)
             if not callable(testFunc):
                 return False
-            fullName = f'%s.%s.%s' % (
-                testCaseClass.__module__, testCaseClass.__qualname__, attrname
-            )
+            fullName = f'{testCaseClass.__module__}.{testCaseClass.__qualname__}.{attrname}'
```

Comment:
I don't think this change should be mixed in with the others. I suggest removing the "f", and no other changes. Changing from %-formatting to f-strings should be a separate change, since it requires a different mindset when reviewing.

Reply:
Ah sorry for this. Ok, I'll separate them two. It will also be a chore I can do in a follow-up PR. Sorry for the noise.